docs/detections/prebuilt-rules/rule-details/unusual-linux-network-connection-discovery.asciidoc

```diff
@@ -105,6 +105,18 @@ Uncommon user command activity can be due to an engineer logging onto a server i
 
 *Framework*: MITRE ATT&CK^TM^
 
+* Tactic:
```

Comment:
If additional Mitre metadata is added, does that require a `Formatting only` version bump? There is no version bumps on the rules with Mitre changes.

Reply:
these versions reflect the rule versions themselves. Mitre threat data will bump rule versions